make/tools.mk

```diff
@@ -49,13 +49,13 @@ PHP                ?= $(X_PHP_TOOL)
 PKG_CONFIG         ?= $(X_PKG_CONFIG)
 
 # Define tool variables for host build
-HOST_CC            ?= $(CC)
-HOST_CXX           ?= $(CXX)
-HOST_AS            ?= $(AS)
-HOST_AR            ?= $(AR)
-HOST_LD            ?= $(LD)
-HOST_PHP           ?= $(PHP)
-HOST_PKG_CONFIG    ?= $(PKG_CONFIG)
+HOST_CC            ?= $(X_CC_TOOL)
```

Comment:
I went a bit further here and decided to use native `$(CC)` and other variables when `CROSS_COMPILE` is not set to 1. Please check latest changes and the case that we can omit setting compiler in `README.md`.

Reply:
I'm getting `HOST_CC=clang` and `HOST_CXX=clang++` even if I explicitly set something like `PLATFORM=Linux` and can't quite figure out why.

Reply:
oof this is the problem/fix (add the single spaces):

```
diff --git a/make/tools.mk b/make/tools.mk
index 33070b38..be2a39e2 100644
--- a/make/tools.mk
+++ b/make/tools.mk
@@ -27,14 +27,14 @@ ifeq ($(PLATFORM),OpenBSD)
   X_AR_TOOL          := ar
   X_LD_TOOL          := ld
   X_PKG_CONFIG       := pkg-config
-else ifeq($(PLATFORM),BSD)
+else ifeq ($(PLATFORM),BSD)
   X_CC_TOOL          := clang
   X_CXX_TOOL         := clang++
   X_AS_TOOL          := as
   X_AR_TOOL          := ar
   X_LD_TOOL          := ld
   X_PKG_CONFIG       := pkg-config
-else ifeq($(PLATFORM),MacOS)
+else ifeq ($(PLATFORM),MacOS)
   X_CC_TOOL          := clang
   X_CXX_TOOL         := clang++
   X_AS_TOOL          := as
```

Reply:
Oh, wow. OK. Thanks for figuring it out, I'll fix it soon.
For Linux the preferred compiler should be `GCC` as it is native compiler. For FreeBSD/MacOS the default compiler should be `clang`.